css/style.css

```diff
@@ -47,31 +47,137 @@ Developer @GA in NYC
 
 body {
      color: #000;
-     font-size: 12px;
      line-height: 1.4;
-     font-family: Helvetica, Arial, sans-serif;
-     background: url(../images/header_bg.jpg) no-repeat center top rgb(134,141,157);
-      background-size: cover;
+     font-family: 'Open Sans', sans-serif;
 }
 
-
 /******************************************
 /* LAYOUT
 /*******************************************/
 
 /* Center the container */
-.container {
-  width: 960px;
-  margin: auto;
-  height: 500px;
+#container {
+  width: 1024px;
+  margin: 0 auto;
 }
 
-header {
+.header {
+  background: url("../images/header_bg.jpg") no-repeat center top rgb(134,141,157);
+  background-size: cover;
+  height: 595px;
+}
 
+.relaxr {
+  font-weight: extra bold;
```

Comment:
@eloisebarrow you can only chose one value, however `extra` is not valid. I think you might be looking for `bolder` 🔨